.github/workflows/compile.yml

```diff
@@ -20,7 +20,7 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
what happened 👀

Reply:
https://twitter.com/onevcat/status/1757787865793413470 It's said that `macos-14` is significantly faster.

But our CI runs out of the memory. Maybe we should allocate more to the node job.

Reply:
try a bigger `--max_old_space_size`?